layouts/_default/baseof.html

```diff
@@ -3,6 +3,7 @@
 
 <head>
     {{- partial "head.html" . }}
+    {{- partial "head_custom_content.html" -}}
```

Comment:
Can you put this in head.html on L#48

Reply:
and rename it to `extend_head.html`

Reply:
> Can you put this in head.html on L#48

and

> and rename it to `extend_head.html`

Done.  Plus added context . to partial call and changed slightly message in the extend_head.html placeholder file since it's content will not be added to the very end of <head> section any more.